app/utils/event_emitter.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+function isFunction(obj: Function): boolean {
+    return typeof obj === 'function';
+}
+
+class EventEmitter {
```

Comment:
Probably out of scope for this, but it would be really nice to have some parity with an existing `Event` reference, i.e. https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent

Reply:
This file was meant to be removed from this PR.  I removed it now :)